docker-compose.spark.yml

```diff
@@ -0,0 +1,56 @@
+version: '2'
+services:
+  spark-master:
+    image: spark-master
+    build:
+      context: ./worker-tasks
+      dockerfile: Dockerfile
+    image: quay.io/azavea/spark:1.6.2
+    restart: always
+    ports:
+      - "7077:7077"
+      - "8888:8080"
+    environment:
+      SPARK_LOCAL_DIRS: "/tmp"
+      SPARK_DAEMON_MEMORY: "256m"
+      SPARK_DAEMON_JAVA_OPTS: "-Dspark.deploy.recoveryMode=FILESYSTEM -Dspark.deploy.recoveryDirectory=/spark-state"
+    volumes:
+      # Spark cluster state
+      - /spark-state
+      # Spark scratch space
+      - /tmp
+    entrypoint: spark-class
+    command: org.apache.spark.deploy.master.Master
+
+  spark-worker:
```

Comment:
Can we expose port 8081 for spark-workers? The Spark Master UI redirects there to view job information in the browser.